web/beacon-app/src/components/ui/WelcomePage/OnboardingHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import { memo } from 'react'
+
+function OnboardingHeader() {
+  return (
+    <main className="mx-auto max-w-4xl mt-16">
+        <h1 className="text-6xl text-center font-bold">Let's Get Eventing</h1>
+        <div className="text-2xl">
+          <p className="mt-8 mx-auto">
+            You did it! What's next? Set up your tenant. Invite team members. Create your first project and get streaming. Update your resume to let everyone know you're a microsrevices expert now.
```

Comment:
Tiny typo here:
microsrevices => microservices

Reply:
Thank you for catching that!